check-peps.py

```diff
@@ -303,6 +303,8 @@ def _validate_discussions_to(line_num: int, line: str) -> MessageIterator:
     """'Discussions-To' must be a thread URL"""
 
     yield from _thread(line_num, line, "Discussions-To", discussions_to=True)
+    if line == "TBA":
```

Comment:
To avoid the inevitable confusion on which initialism it is:
```suggestion
    if line in {"TBA", "TBD"}:
```